pensjon-brevbaker/src/main/kotlin/no/nav/pensjon/etterlatte/maler/tilbakekreving/Tilbakekreving.kt

```diff
@@ -0,0 +1,163 @@
+            displayTitle = "Vedtak - Tilbakekreving",
+            isSensitiv = true,
+            distribusjonstype = LetterMetadata.Distribusjonstype.VEDTAK,
+            brevtype = LetterMetadata.Brevtype.VEDTAKSBREV,
+        ),
+    ) {
+        title {
+            text(
+                Bokmal to "Du må betale tilbake gjenlevendepensjon",
+                Nynorsk to "TODO nynorsk",
+                English to "TODO engelsk"
+            )
+        }
+        outline {
+            includePhrase(TilbakekrevingInnhold)
+            includePhrase(DuHarRettTilAaKlage)
+            includePhrase(DuHarRettTilInnsyn)
+            includePhrase(HarDuSpoersmaal(erOMS, erBP))
+        }
+    }
+}
+
+private object TilbakekrevingInnhold : OutlinePhrase<LangBokmalNynorskEnglish>() {
+    override fun OutlineOnlyScope<LangBokmalNynorskEnglish, Unit>.template() {
+        paragraph {
+            text(
+                Bokmal to """
+                    HER KOMMER INNHOLD TIL TILBAKEKREVINGSBREV
+               """.trimIndent(), // TODO
+                Nynorsk to "",
+                English to ""
+            )
+        }
+    }
+}
+
+private object DuHarRettTilAaKlage : OutlinePhrase<LangBokmalNynorskEnglish>() {
+    override fun OutlineOnlyScope<LangBokmalNynorskEnglish, Unit>.template() {
+        title2 {
+            text(
+                Bokmal to "Du har rett til å klage",
+                Nynorsk to "",
+                English to ""
+            )
+        }
+        paragraph {
+            text(
+                Bokmal to """
+Hvis du mener at vedtaket er feil, kan du klage innen seks uker fra den datoen du mottok dette vedtaket.  
+Klagen kan sendes via innlogging på nettsiden vår nav.no, eller sendes til oss i posten. 
+Skriftlig klage som sendes i posten må inneholde navn, fødselsnummer og adresse, og den må være underskrevet av deg.
+Bruk gjerne skjemaet som du finner på ${Constants.KLAGE_URL}.  
+Klagen sendes til ${Constants.POSTADRESSE}.
+                """.trimIndent(),
+                Nynorsk to "",
+                English to ""
+            )
+        }
+        paragraph {
+            text(
+                Bokmal to "RESTERENDE FOR KLAGE KOMMER", // TODO
+                Nynorsk to "",
+                English to "",
+            )
+        }
+
+    }
+}
+
+private object DuHarRettTilInnsyn : OutlinePhrase<LangBokmalNynorskEnglish>() {
+    override fun OutlineOnlyScope<LangBokmalNynorskEnglish, Unit>.template() {
+        title2 {
+            text(
+                Bokmal to "Du har rett til innsyn",
+                Nynorsk to "",
+                English to ""
+            )
+        }
+        paragraph {
+            text(
+                Bokmal to """
+På nav.no/Min side kan du se dokumentene i saken din. 
+               """.trimIndent(),
+                Nynorsk to "",
+                English to ""
+            )
+        }
+    }
+}
+
+private data class HarDuSpoersmaal(
```

Comment:
Det finnes ikke en HarDuSpoersmaal i felles?

Reply:
Jo flere. Men Liv Inger har skrevet en ny tekst for det som var ulik de andre.